students/BrianNisonger/Lesson4/test_customer_db.py

```diff
@@ -0,0 +1,170 @@
+from basic_operations import add_customer
+from basic_operations import search_customer
+from basic_operations import delete_customer
+from basic_operations import update_customer
+from basic_operations import list_active_customers
+
+from customer_model import Customer
+import pytest
+
+ok_customer = {
+    'customer_id': 'W3434fd',
+    'first_name': 'Suzie',
+    'last_name': 'Edgar',
+    'home_address': '123 Browncroft Blvd, Rochester, NY 97235',
+    'phone_number': '234-123-4567',
+    'email_address': 'suzie@hotmail.com',
+    'status': True,
+    'credit_limit': '40'
+}
+
```

Comment:
Looks like you are not creating/deleting a database for your tests. I highly recommend doing this. It is very important to understand how to differentiate test databases from prod, and to learn how to use fixtures in tests.